boards/arm/stm32f103_mini/stm32f103_mini.dts

```diff
@@ -33,18 +33,23 @@
 };
 
 &clk_hsi {
+	status = "disabled";
+};
```

Comment:
Default status for clk_hsi node is disabled (in dtsi file).
So there is no value keeping this node with disabled status, please remove it.